BoostTestItemTemplate/BoostTest.vstemplate

```diff
@@ -1,7 +1,7 @@
 <VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Item">
   <TemplateData>
     <Name>Boost.Test</Name>

```

Comment:
This needs to be localizable too. For example in French I'm seeing "Projet de test MSTest (.NET Core)" instead of "MSTest Test"

Reply:
Done!